scripts/format-check.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+const colors = require('colors/safe');
+const path = require('path');
+const prettier = path.join(process.cwd(), 'node_modules', '.bin', 'prettier');
+const shell = require('shelljs');
+const unPrettyFiles = [];
+
+console.log('Formatting Test');
+
+let result = shell.exec(`${prettier} --list-different ./**/*.{js,json,md,ts,scss}`, { silent: true });
```

Comment:
Do we really need to have a script for this? Looks like `--list-different` will error out for you. https://prettier.io/docs/en/cli.html#list-different

Reply:
Right, but it won't give us a list of unformatted files (so I quickly know where to look when a push fails).

Reply:
Actually, my last comment is incorrect, yes it will give us a list, but I'd still like to keep the script so I can see a count and colors for quickly scanning the cli output on my local. 
\cc @gnomeontherun